### ISSUE TITLE
Final Report for Cindy

### ISSUE THREAD

**Issue by SilverMagnum** [CampConn/Capstone#24]
### The Final Issue: Questions to Answer

1. Which implementation worked best?
2. How much more effective was it than other implementations?
3. Why did each implementation do as well or poorly as they did?
4. Could other implementations prove more effective?
5. Is more training data required?
6. How practically can these methods be run in real time?
7. How effectively can methods be generalized? That is test each model on each test set.
